packages/webdriver/src/types.ts

```diff
@@ -260,12 +260,13 @@ export interface W3CCapabilities {
     firstMatch: Capabilities[];
 }
 
-export interface DesiredCapabilities extends Capabilities, SauceLabsCapabilities, SeleniumRCCapabilities, AppiumIOSCapabilities, GeckodriverCapabilities, IECapabilities, AppiumAndroidCapabilities, AppiumCapabilities, VendorExtensions, GridCapabilities, ChromeCapabilities {
+export interface DesiredCapabilities extends Capabilities, SauceLabsCapabilities, TestingbotCapabilities, SeleniumRCCapabilities, AppiumIOSCapabilities, GeckodriverCapabilities, IECapabilities, AppiumAndroidCapabilities, AppiumCapabilities, VendorExtensions, GridCapabilities, ChromeCapabilities {
     // Read-only capabilities
     cssSelectorsEnabled?: boolean;
     handlesAlerts?: boolean;
     version?: string;
     platform?: string;
+    public?: any;
```

Comment:
`SauceLabsCapabilities` and `TestingbotCapabilities` has conflicting types of property `public`, so extending pattern will not work here. Casting public to any is the only workaround for now. I think we should use a different typing pattern in the future.

Reply:
Let's make Sauce Labs property a boolean too and get rid of this.

Reply:
But it is indeed string, possible values are public, public restricted, share and team
https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options